javascript/javascript2/week2/hyfBayHelpers.js

```diff
@@ -0,0 +1,72 @@
+/* DONT MODIFY ANY OF THIS CODE!!!*/
+
+window.getAvailableProducts = function () {
```

Comment:
function inside a function will increase 'javascript cognitive complexity. For this exercise it is fine but once you have time please check
google "javascript cognitive complexity' and try to achieve same task by object-oriented JS
https://codepen.io/naresh699/pen/yoZZJe